pages/common/git-worktree.md

```diff
@@ -0,0 +1,15 @@
+# git worktree
+
+> Manage multiple working trees attached to the same repository.
+
+- Create a new folder with the specified branch checked out into it:
+
+`git worktree add {{path/to/folder}} {{branch}}`
```

Comment:
> In this case, we should put `add` first, then `list` and `prune` at last.

I think he was asking whether `add -b` should be grouped with plain `add`, or moved down for being a less common example. I think this call is often subjective so there isn't a single answer, but in this case I'd say it makes more sense for both `add` examples to be together.